src/index.html

```diff
@@ -2,16 +2,254 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+    <form 
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      class="form1" 
+      id="form" 
+      method="post"
+    >
+      <fieldset class="fieldset one">
+        <legend>Personal information</legend>
+        <label 
+          for="sname" 
+          class="label"
+        >
+          Surname:
+          <input 
+            type="text"
+            id="sname" 
+            name="sname" 
```

Comment:
https://github.com/mate-academy/layout_html-form/pull/1478#discussion_r827800888
@PRO-GRAM-MER Paul did comment already - better use longed, but more descriptive names.
```suggestion
            id="surname" 
            name="surname" 
```